middleware/http/oauth2/oauth2_middleware.go

```diff
@@ -32,15 +32,19 @@ import (
 )
 
 // Metadata is the oAuth middleware config.
+// For RedirectURLWhitelist, Each URL can be a wildcard, e.g. https://*.dapr.io . Empty whitelist means no restriction.
+// Please be careful about using wildcards and empty whitelist in production environments, as this may be a security risk.
 type oAuth2MiddlewareMetadata struct {
-	ClientID       string `json:"clientID" mapstructure:"clientID"`
-	ClientSecret   string `json:"clientSecret" mapstructure:"clientSecret"`
-	Scopes         string `json:"scopes" mapstructure:"scopes"`
-	AuthURL        string `json:"authURL" mapstructure:"authURL"`
-	TokenURL       string `json:"tokenURL" mapstructure:"tokenURL"`
-	AuthHeaderName string `json:"authHeaderName" mapstructure:"authHeaderName"`
-	RedirectURL    string `json:"redirectURL" mapstructure:"redirectURL"`
-	ForceHTTPS     string `json:"forceHTTPS" mapstructure:"forceHTTPS"`
+	ClientID             string   `json:"clientID" mapstructure:"clientID"`
+	ClientSecret         string   `json:"clientSecret" mapstructure:"clientSecret"`
+	Scopes               string   `json:"scopes" mapstructure:"scopes"`
+	AuthURL              string   `json:"authURL" mapstructure:"authURL"`
+	TokenURL             string   `json:"tokenURL" mapstructure:"tokenURL"`
+	AuthHeaderName       string   `json:"authHeaderName" mapstructure:"authHeaderName"`
+	RedirectParamName    string   `json:"redirectParamName" mapstructure:"redirectParamName"`
+	RedirectURL          string   `json:"redirectURL" mapstructure:"redirectURL"`
+	RedirectURLWhitelist []string `json:"redirectURLWhitelist" mapstructure:"redirectURLWhitelist"`
```

Comment:
Please change whitelist to allowlist, that's the terminology we use for this feature in other Dapr building blocks such as service invocation

Reply:
resolved